pkg/tasks/create.go

```diff
@@ -222,3 +222,11 @@ func CreateTask(ctx context.Context, kubeClient client.Client, command controlap
 
 	return task, nil
 }
+
+func DcNames(dcName string) []string {
```

Comment:
Why is this exported? Also, instead of modifying each task, would it have made more sense to modify the "CreateClusterTask()" function instead to modify this part:

```go
	if len(datacenters) > 0 {
		task.Spec.Datacenters = datacenters
	}
```

To something like

```go
	if len(datacenters) > 0 && datacenters[0] != "" {
		task.Spec.Datacenters = datacenters
	}
```

?

Reply:
> Why is this exported?

For testing purposes.

> Also, instead of modifying each task, would it have made more sense to modify the "CreateClusterTask()" function instead to modify this part:

Done